### PR TITLE
add cv

### PR DESCRIPTION
**Pull request by lgimeneza**

### CODE REVIEW

**Comment by albanelia0** [comment]
Good job! ^^

File: staff/lgimeneza/cv/index.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Curriculum Viae</h1>
+    <h2>L. Giménez</h2>
+    <h3>Datos personales</h3>
+    <ul>
+        <li><h4>Telefono:</h4> 666666666 </li>
+        <li><h4>Direccion:</h4> Calle Mallor nº1 </li>
```
Comment (mikelpmc) [inline staff/lgimeneza/cv/index.html]: *Mayor

5113847621_10e4cbdb2e_b

**Review by mikelpmc — APPROVED** [review]
Very good 👍

**Comment by lgimeneza** [comment]
Done! Thanks

**Pull request merged**